WordPress/Classes/Utility/WPTableViewHandler.h

```diff
@@ -97,4 +96,18 @@
 - (void)refreshCachedRowHeightsForWidth:(CGFloat)width;
 - (void)invalidateCachedRowHeightAtIndexPath:(NSIndexPath *)indexPath;
 
+/**
+ A convenience method for clearing cached row heights and reloading the table view.
+ */
+- (void)refreshTableView;
+
+/**
+ Reloads the table, adjusting the table view's content offset so that the currently
+ visible content stays in place.
+ 
+ The caller should update the tableview's content, (i.e. the fetched results)  
+ in the `tableViewHandlerWillRefreshTableViewPreservingOffset:` delegate method.
+ */
+- (void)refreshTableViewPreservingOffset;
```

Comment:
(Sounds also like a good Unit Testing candidate, but i'm also not sure how to mock up UITableView!!)